pkgs/by-name/ku/kurva/package.nix

```
@@ -0,0 +1,36 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  nix-update-script,
+  kdePackages,
+}:
+stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```

Prefer finalAttrs here as this reduces evaluation time.

---

Ah, I remember reading this from somewhere but this isn't blocking anyways. But I would say use whatever you prefer. I just prefer finalAttrs